controller/volume_controller.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	batchv1 "k8s.io/api/batch/v1"
 	batchv1beta1 "k8s.io/api/batch/v1beta1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
?

Reply:
`go fmt` tool of Goland will automatically add it...